pl_examples/domain_templates/imagenet.py

```diff
@@ -86,53 +80,56 @@ def __init__(
         self.workers = workers
         self.model = models.__dict__[self.arch](pretrained=self.pretrained)
 
+        self.train_acc1 = Accuracy(top_k=1)
+        self.train_acc5 = Accuracy(top_k=5)
+        self.eval_acc1 = Accuracy(top_k=1)
+        self.eval_acc5 = Accuracy(top_k=5)
+
     def forward(self, x):
         return self.model(x)
 
     def training_step(self, batch, batch_idx):
         images, target = batch
         output = self(images)
```

Comment:
```suggestion
        output = self.model(images)
```
just something we recommend to users.